src/ui/projects/api.py

```diff
@@ -449,3 +449,37 @@ def check_api(self) -> bool:
             return False
 
         return True
+
+    def get_objects(
+        self, project_id: int, job_id: int, start: int, lenght: int
```

Comment:
```suggestion
        self, project_id: int, job_id: int, start: int, length: int
```

Reply:
This PR is based upon #72 and this is from there. It is all ready fixed in that PR.